room_assistance/indico_room_assistance/forms.py

```diff
@@ -62,6 +74,8 @@ def __init__(self, *args, **kwargs):
 
     def validate_occurrences(self, field):
         for req_dt in field.data:
+            if type(req_dt) is date:
```

Comment:
`isinstance()`

Reply:
and i'd rather do `not isinstance(req_dt, datetime)`

Reply:
The same as my response above.

Reply:
I think here you *can* safely  use the isinstance check since you really want a `datetime`, and that one is not a subclass of date nor time